test/ImpactEvaluator.t.sol

```diff
@@ -14,14 +14,26 @@ contract ImpactEvaluatorTest is Test {
             1000
         );
         assertEq(impactEvaluator.currentRoundIndex(), 0);
-        assertNotEq(impactEvaluator.getRound(0).start, 0);
-        assertEq(impactEvaluator.getRound(0).start, block.number);
+        assertNotEq(impactEvaluator.getRound(0).end, 0);
+        assertEq(impactEvaluator.getRound(0).end, block.number + 1000);
```

Comment:
What is the `1000` here referring to?

Reply:
Here we're asserting that the end for the current round was set 1000 blocks in the future, which is correct according to the round length set in the constructor